docs/dev/run-cvo-locally.md

```diff
@@ -0,0 +1,51 @@
+# How to Build and Run CVO Locally
+
+This document provides the instructions to build and run the CVO executable locally. Local execution of CVO, for example executing CVO on your development laptop, allows a "quick-turn" test capability whereby the CVO executable can be quickly rebuilt with changes and launched again without the need of starting a new OCP cluster.
+
+## Preconditions
+
+Build the CVO executable using the standard procedure:
+
+```console
+$ cd ~/go/src/github.com/openshift/cluster-version-operator
+$ make
+```
+
+Start an OCP cluster of the appropriate version. One that is compatiable with the version of CVO with which you're working.
+
+Extract and download the OCP cluster release:
+
+```console
+$ oc image extract -a ~/pull-secret-internal quay.io/openshift-release-dev/ocp-release:4.4.0-rc.4-x86_64 --path /:/tmp/release/
+```
+
+## Steps to Run CVO Locally
+
+Scale down the CVO pod on the cluster:
+
+```console
+$ oc scale --replicas=0 deployment.apps/cluster-version-operator
+```
+
+Set the following environment variables used by CVO:
+
+```console
+# Set to anything
+$ export NODE_NAME=foobar
+
+# Set to release extracted above
+$ export PAYLOAD_OVERRIDE=/tmp/release/
+
+# Point to cluster
+$ export KUBECONFIG=~/Downloads/cluster-bot-2020-05-29-183515.kubeconfig
```

Comment:
This step should be before `oc scale --replicas=0 deployment.apps/cluster-version-operator` as you need access to the cluster for scaling down the CVO.

Reply:
The importance of this step, in the context of this document, is that it is required to run CVO locally so I left it in this section and modified the above paragraph (in [3bdbbc0](https://github.com/openshift/cluster-version-operator/pull/381/commits/3bdbbc07d01ff7c4374c039cb68a84564bea7abd)) instructing the reader to login to the cluster. That is also my typical work flow when using cluster-bot.

Reply:
@jottofar If the user had already logged in to the cluster in step `Login to the cluster using `oc` why does user again need to set Kubeconfig environment variable? Not sure why you do this in your workflow. May be because you do this in a separate terminal windows. 

 I have tested the steps in this document and it worked for me without this step because I had already logged in to the cluster using KUBECONFIG env variable in the beginning to scale down the CVO.

Reply:
@jottofar If the user had already logged in to the cluster in step Login to the cluster using oc` why does user again need to set Kubeconfig environment variable? Not sure why you do this in your workflow. May be because you do this in a separate terminal windows.

I have tested the steps in this document and it worked for me without this step because I had already logged in to the cluster using KUBECONFIG env variable in the beginning to scale down the CVO.